thoth/package_extract/core.py

```diff
@@ -47,20 +50,31 @@ def extract_buildlog(input_text: str) -> typing.List[dict]:
 def extract_image(image_name: str, timeout: int = None, *, registry_credentials: str = None,
                   tls_verify: bool=True) -> dict:
     """Extract dependencies from an image."""
-    image_name = quote(image_name)
-    with tempfile.TemporaryDirectory() as dir_path:
-        download_image(
-            image_name,
-            dir_path,
-            timeout=timeout or None,
-            registry_credentials=registry_credentials or None,
-            tls_verify=tls_verify
-        )
+    """Begin the timer for when the job starts"""
+    with _METRIC_ANALYZER_JOB.time():
+            image_name = quote(image_name)
+            with tempfile.TemporaryDirectory() as dir_path:
+                download_image(
+                    image_name,
+                    dir_path,
+                    timeout=timeout or None,
+                    registry_credentials=registry_credentials or None,
+                    tls_verify=tls_verify
+                )
 
-        rootfs_path = os.path.join(dir_path, 'rootfs')
-        layers = construct_rootfs(dir_path, rootfs_path)
+                rootfs_path = os.path.join(dir_path, 'rootfs')
+                layers = construct_rootfs(dir_path, rootfs_path)
 
-        result = run_analyzers(rootfs_path)
-        result['layers'] = layers
+                result = run_analyzers(rootfs_path)
+                result['layers'] = layers
 
-        return result
+                return result
+        
+        push_gateway = os.getenv('PROMETHEUS_PUSH_GATEWAY', 'pushgateway:9091')
```

Comment:
I'm not that fine with the default value - ``pushgateway:9091``. This works for our current test cluster, but I would rather see it explicit (provided by env vars in deployment templates). Feel free to keep it for now, but IMHO this should be also changed in the ``package-releases-job``.

Reply:
Okay, will keep this in mind.